internal/eval/evalers_test.go

```diff
@@ -1860,10 +1861,10 @@ func TestEntityIn(t *testing.T) {
 
 		entityMap := types.Entities{}
 		for i := 0; i < 100; i++ {
-			p := []types.EntityUID{
+			p := sets.NewMapSetFromSlice([]types.EntityUID{
```

Comment:
Maybe if the package was called `mapset` you could just do `mapset.New(variadic...)`